### ISSUE TITLE
Restructuring and Reducing the CEA database (up for discussion)

### ISSUE THREAD

**Issue by shanshanhsieh** [architecture-building-systems/CityEnergyAnalyst#2553]
Here are some points we discussed in the CEA meeting (6th Feb), we still need to make the final calls in the coming days.
@martin-mosteiro @daren-thomas @shizhongming any thought is welcome!

- [x] **1.** Remove Primary Energy everywhere (the behavior is linear to emission)
- [x] This information is currently linked to: optimization, life-cycle analysis, feedstocks, mobility, embodied energy.
- [x] @martin-mosteiro , are you using PE as a performance indicator?
- [x] **2.** Move LIFECYCLE into ARCHETYPES as an EMISSION FACTOR tab
- [x] **3.** Add a few tabs to the **input database** that allows the user to modify easily. 
- [x] - New tab: "EMISSION INTENSITY" (see point 2)
- [ ] **4.** Clean up the columns inside the system database
- [ ] - USD-2015
- [ ] - remove Type (in the "PV" tab)
- [x] - changes in these tab involve many changes in the scripts
- [x] **5.** Add information about cost calculation to "supply_systems" tab
- [ ] **6.** Add a yaml file to specify in which tab a new row can be added.
- [ ] - cannot: "cooling" and "heating" tabs under air_conditioning_systems
- [ ] **7.** Validate input values in each cell 
- [ ] - some columns can only contain unique value
- [ ] - some cells can only take a limited number of values (e.g. T1, T2, T3)
- [x] **8.** where too keep:  "FEEDSTOCK",  "ALL_IN_ON_SYSTEM"?
- [x] move feedstocks to a separate spreadsheet and combine it with detailed electricity costs. 24 hour input so the optimisation can use different hourly tariffs during the day. Emission factors can also be an opportunity.

**Comment by martin-mosteiro** [comment]
1. What do you mean by PEN being linear to emission?
Here's what the CH feedstocks in CEA look like:
![image](https://user-images.githubusercontent.com/18548065/73940344-960a6700-48eb-11ea-8b13-096a6f82ca31.png)
The Swiss grid is a huge outlier because it's low emission but highly PEN intensive (due to nuclear).

Here's what it looks like for SG:
![image](https://user-images.githubusercontent.com/18548065/73940673-2648ac00-48ec-11ea-949e-90f89ca12a10.png)

Bottom line: I do use PEN as an indicator, but I'm not really using CEA for any of these calculations, so it doesn't matter to me much.

**Comment by martin-mosteiro** [comment]
Everything else sounds okay to me. Including keeping "FEEDSTOCK" and "ALL_IN_ONE_SYSTEM" where they are - might not be ideal, but they're relevant for supply systems, so that looks okay to me.

**Comment by JIMENOFONSECA** [comment]
@reyery there are several points we already cover in the PR #2557 , now there are some issue we have in terms of validation. You are on top of that right?

**Comment by JIMENOFONSECA** [comment]
closing, new separate issues are created for the remaining work
